content/en/logs/guide/access-your-log-data-programmatically.md

```diff
@@ -0,0 +1,524 @@
+
+- An API key and an application key from an admin user is required to use the API. These are available in your [Datadog account API key page][2]. Replace `<DATADOG_API_KEY>` and `<DATADOG_APP_KEY>` with your Datadog API key and your Datadog application key.
+
+- This guide features `curl` examples. Install [curl][3] if you do not have it installed, or reference additional language examples for this API endpoint in the [API documentation][1].
+
+## Examples
+
+### Basic search
+
+To retrieve all log events within a specific time period, use the following [Search Syntax][4] to complete the API call.
+
+`from` indicates the `start time` and `to` indicates the `end time` for the log data. `query` indicates the search query that must be executed.
+
+**API call:**
+
+```bash
+
+curl -L -X POST "https://api.{{< region-param key="dd_site" code="true" >}}/api/v2/logs/events/search" -H "Content-Type: application/json" -H "DD-API-KEY: <DATADOG_API_KEY>" -H "DD-APPLICATION-KEY: <DATADOG_APP_KEY>" --data-raw '{
+  "filter": {
+    "from": "2020-10-07T00:00:00+00:00",
+    "to": "2020-10-07T00:15:00+00:00",
+    "query": "*"
+  }
+}'
+
+```
+
+**Response:**
+
+The result dataset is comprised of the `data` object, as depicted in the following example response.
+
+```json
+
+{
+    "meta": {
+        "page": {
+            "after": "eyJhZnRlciI6IkFRQUFBWFVBWFJZVkpHeHZEUUFBQUFCQldGVkJXRkpaVm1kMlpsa3RiVWRVWmpSQlFRIn0"
+        }
+    },
+    "data": [
+        {
+            "attributes": {
+                "status": "info",
+                "service": "pageViewService",
+                "tags": [
+                    "source:postman",
+                    "project:test"
+                ],
+                "timestamp": "2020-10-07T00:02:33.461Z",
+                "host": "my.sample.host",
+                "attributes": {
+                    "hostname": "my.sample.host",
+                    "pageViews": "450",
+                    "user": "chris",
+                    "service": "pageViewService"
+                },
+                "message": "Sample message"
+            },
+            "type": "log",
+            "id": "AQAAAXUAXd91M9wyTgAAAABBWFVBWGQ5MVZrQmFzdEZ2TG9BQQ"
+        },
+        {
+            "attributes": {
+                "status": "info",
+                "service": "pageViewService",
+                "tags": [
+                    "source:postman",
+                    "project:test"
+                ],
+                "timestamp": "2020-10-07T00:01:57.586Z",
+                "host": "my.sample.host",
+                "attributes": {
+                    "hostname": "my.sample.host",
+                    "pageViews": "500",
+                    "user": "bob",
+                    "service": "pageViewService"
+                },
+                "message": "Sample message"
+            },
+            "type": "log",
+            "id": "AQAAAXUAXVNSvuMvWwAAAABBWFVBWFZOU2I2ZWcxX3c2LVVBQQ"
+        },
+        {
+            "attributes": {
```

Comment:
Let's keep only 2 logs in the sample result for the sake of compacity of the page.